src/App.js

```diff
@@ -1,7 +1,31 @@
 import "./App.css";
+import productData from "./data/productData"
+import React, { Component } from 'react'
+import Products from "./components/Products"
+import ProductCard from "./components/ProductCard";
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
+export default class App extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      
+    }
+  }
+  render() {
+    const {students} = this.props
+    return (
+      <div className="Products">
+        <h3>My Garage Sale</h3>
+        
+        <ProductCard
+        students = {students}
+         saleItems={productData}
+         />
```

Comment:
I see here you're rendering ProductCard and passing it two props. But in ProductCard itself, you aren't referencing either `students` or `saleItems`. 

In `Products.js` you are using `students` as a prop and mapping over it, but I don't see it rendered anywhere!